### PR TITLE
releng: Update bot comment for patch releases

### PR DESCRIPTION
**Pull request by justaugustus**
Update patch release bot comment to reference @kubernetes/release-managers, as we'll be consolidating the Patch Release Team and Branch Managers into one team.

Signed-off-by: Stephen Augustus <saugustus@vmware.com>

/hold until https://github.com/kubernetes/sig-release/pull/1106 merges
/assign @tpepper 
cc: @kubernetes/release-engineering

### CODE REVIEW

File: config/prow/plugins.yaml [hunk 1]
```diff
@@ -537,11 +537,14 @@ retitle:
 
 cherry_pick_unapproved:
   comment: |
-    This cherry pick PR is for a release branch and has not yet been approved by the Patch Release Team.
+    This cherry pick PR is for a release branch and has not yet been approved by [Release Managers](https://git.k8s.io/sig-release/release-managers.md).
     Adding the `do-not-merge/cherry-pick-not-approved` label.
```
Comment (saschagrunert) [inline config/prow/plugins.yaml:541]: Just out of curiosity: do you think we can switch over to the usual `/lgtm` and `/approve` workflow without having a need for this additional label in the future?

Comment (justaugustus) [reply config/prow/plugins.yaml:541]: @saschagrunert -- We still need patch review from a Release Manager to vet that a PR is actually appropriate for cherry pick.

Comment (saschagrunert) [reply config/prow/plugins.yaml:541]: Okay thank you for the clarification. :)

**Comment by tpepper** [comment]
/lgtm
/approve

and still
/hold
for the set of associated changes outlined in https://github.com/kubernetes/sig-release/pull/1106

**Comment by justaugustus** [comment]
Thanks for the reviews everyone!
/hold cancel

File: config/prow/plugins.yaml [hunk 2]
```diff
@@ -537,11 +537,14 @@ retitle:
...
 
-    To merge this cherry pick, please ping the **kubernetes/patch-release-team** in a comment, *after* it has been approved by the relevant OWNERS.
-    For details on the patch release process and schedule, see the [Patch Releases](https://git.k8s.io/sig-release/releases/patch-releases.md) page.
+    To merge this cherry pick, it must first be approved (`/lgtm` + `/approve`) by the relevant OWNERS.
+
+    **AFTER** it has been approved by code owners, please ping the **kubernetes/release-managers** in a comment to request a cherry pick review.
```
Comment (spiffxp) [inline config/prow/plugins.yaml:545]:
```suggestion
    **AFTER** it has been approved by code owners, please ping the **kubernetes/release-managers** team in a comment to request a cherry pick review.
```

**Pull request merged**